### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR updates all the modules and the relevant subworkflows.

Main changes are the removal of *no_meta channels for fasta, fai, and sequence dictionary files.

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/annotation/annotate_cadd.nf [hunk 1]
```diff
@@ -32,10 +32,10 @@ workflow ANNOTATE_CADD {
             .join(ch_index)
             .join(CADD.out.tsv)
             .join(TABIX_CADD.out.tbi)
-            .groupTuple()
+            .combine(ch_header)
```
Comment (jemten) [inline subworkflows/local/annotation/annotate_cadd.nf:35]: 👍

**Review by jemten — APPROVED** [review]
Nice cleanup!

**Comment by Lucpen** [comment]
Yeah, very nice :D

**Review by Lucpen — APPROVED** [review]
LGTM

**Pull request merged**